### ISSUE TITLE
Accounting by Tier Level

### ISSUE THREAD

**Issue by graciani** [DIRACGrid/DIRAC#291]
It would be nice that the Sites menu of the Accounting Allows extra keywords "Tier1", "Tier2",... The tier level is no used to sort the list of Sites but a explicit way to select all of them at a given level.

**Comment by fstagni** [comment]
5 years later, no progress nor interest. So, closing.

**Comment by phicharp** [comment]
This is high on my wishlist as well as that of Concezio I am sure! 
So there is interest and it should just be assigned to someone! But as only very few people can assign, and this was set before Zoltan was responsible, it escaped... I would propose to reopen and assign it to Zoltan. I think this is a must in addition to:

* Also for site selection ,selecting by Grid (LCG, DIRAC, Cloud, VAC...) would be nice
* When selecting storage elements, it would be good to also be able to select SEGroups (just add all SE groups at the top of the list)
Note: a cleanup of the SEs would help ;-)

**Comment by fstagni** [comment]
Alright. Re-opened and assigned to Zoltan.

**Comment by phicharp** [comment]
@zmathe: any comment on the feasibility?

**Comment by zmathe** [comment]
This can be implemented, but I have to see where the information can be retrieved. I can add to the menu for example: Tier1 and when Tier1 menu item will be pressed then all Tier1 sites automatically selected. This can be quite generic and we can use everywhere. Right now I do not see any other possibility.  But suggestion examples are welcome.

**Comment by phicharp** [comment]
OK, let's discuss this in front of a screen then!
Essentially I think that we should use the StorageElementGroups for creating either a separate menu or put at top of the existing menu. Clicking for example on Tier1-BUFFER would then select automatically all SEs in that group.
Similarly for sites selections, we should have several groupings:
1. By Tier level (easy)
2. By "Grid", e.g. LCG, VAC, DIRAC etc...
3. By Country (reminder: CERN should be moved out of Switzerland cern.ch -> cern.cern)

**Comment by roiser** [comment]
Hi, I'd be also interested in this feature. I have one more additional requirement. I.e. I would like to select e.g. all the T2 sites with MoU agreements, so combining two groupings.

**Comment by phicharp** [comment]
Yes, we should just find another name for these as they should not formally be called Tier2s! CMS calls them Tier3 which is _wrong_ but this is a minor detail, we could call them whatever we like (TierNone).
It would be good we tackle finally this question that is now 6 years old!

**Comment by roiser** [comment]
Agree on the timeline ;-)

For what concerns selections, what about defining "tags" in the CS for the different resource items and we then could select e.g. all "T2 sites" in "Europe" which have signed the "MoU" and are led by "administrators with blue eyes" (to give a half stupid example). 

If this is too much work, deriving the necessary info from the current CS entries is also fine, its just a bit more constraint in what we could possibly do in the future.

**Comment by phicharp** [comment]
@zmathe: yesterday you said (almost) all requests for the web portal had been included in the portal 4.0
Does it include that one?

**Comment by zmathe** [comment]
No, this one is not included. I can not add it to the current selector. I have to change the selector to "tree" kind of menu. I increased the priority of this task.

**Comment by phicharp** [comment]
Gentle reminder for this portal enhancement!
